include/aws/http/connection.h

```diff
@@ -241,6 +275,28 @@ struct aws_http_client_connection_options {
      * reaches 0, no further data will be received.
      **/
     bool manual_window_management;
+
+    /**
+     * HTTP/2 specific options.
+     * Optional for HTTP/2 connection. Ignored for HTTP/1 connection.
+     */
+    struct aws_http2_connection_options {
+        /**
+         * Required.
+         * Settings to change as the initial local settings.
+         * If set, these settings will be sent to remote peer once the connection set up.
+         * It can be zero out, no change will made to the default initial settings defined by from RFC-7540 6.5.2.
+         * Note: you may want to disable PUSH here, since we don't support server push for now.
```

Comment:
Or we can initialize it by AWS_HTTP_CLIENT_CONNECTION_OPTIONS_INIT. We probably want an AWS_HTTP2_CLIENT_CONNECTION_OPTIONS_INIT.